quickstarts/microsoft.web/webapp-linux-sonarqube-postgresql-private-vnet/main.bicep

```diff
@@ -0,0 +1,269 @@
+targetScope = 'resourceGroup'
+
+@description('Name of Azure Web App')
+param siteName string
+
+@description('The location into which the resources should be deployed')
+param location string = resourceGroup().location
+
+@description('Database administrator login name')
+@minLength(1)
+@secure()
+param administratorLogin string
+
+@description('Database administrator password')
+@minLength(8)
+@maxLength(128)
+@secure()
+param administratorLoginPassword string
+
+@description('The tier of the particular SKU, e.g. Burstable')
+@allowed([
+  'Burstable'
+  'GeneralPurpose'
+  'MemoryOptimized'
+])
+param postgresFlexibleServersSkuTier string = 'Burstable'
+
+@description('The name of the sku, typically, tier + family + cores, e.g. Standard_D4s_v3')
+@allowed([
```

Comment:
remove allowed values as it will block deployment in subs/regions that don't have the sizes available...